app/features/utils/functions.js

```diff
@@ -5,6 +5,29 @@
 import { shell } from 'electron';
 import config from '../config';
 
+/**
+ * Finds the external_api.js for the given Server URL. It also handles if the
+ * server URL contains tenant name.
+ *
+ * @param {string} serverURL - Jitsi Meet Server URL.
+ * @returns {Promise} - Promise which returns the external_api.js url.
+ */
+export async function findExternalApiURL(serverURL: string) {
```

Comment:
Something is wrong here. If I use some incorrect URL it never returns an error. Try https://saghul.net/foo/bar

Also, for some reason I get a weird result with https://example.com/foo/bar